webapi/tct-text-css3-tests/css3-text-app/testscripts/CSS3TextEffects_hyphens_manual.feature

```diff
@@ -0,0 +1,6 @@
+Feature: css3-text
+ Scenario: CSS3TextEffects hyphens
+   When launch "css3-text-app"
+     And I go to "text/CSS3TextEffects_hyphens_manual.html"
+     And I save the page to "CSS3TextEffects_hyphens_manual"
```

Comment:
No need "manual" tag in feature file here, please follow the rule we talked before, same for feature name